scripts/mintTokens.js

```diff
@@ -97,7 +97,7 @@ const mintTokens = async (contract, authTokens, imageRecords) => {
 
     // eslint-disable-next-line no-await-in-loop
     await axios
-      .post('/users/title-metadata', requestBody, requestOptions)
+      .post('/users/record-metadata', requestBody, requestOptions)
```

Comment:
Does this endpoint need to get renamed too?

Reply:
Yeah, it'll need to be renamed 👍